website/source/docs/agent/telemetry.html.md

```diff
@@ -462,6 +462,30 @@ page for more details.
     <td>Integer</td>
     <td>Gauge</td>
   </tr>
+  <tr>
+    <td>`nomad.client.allocs.<Job>.<TaskGroup>.<AllocID>.<Task>.cpu.allocated`</td>
+    <td>Total CPU allocated for this task</td>
+    <td>Integer</td>
+    <td>Gauge</td>
+  </tr>
+  <tr>
+    <td>`nomad.client.allocs.<Job>.<TaskGroup>.<AllocID>.<Task>.memory.allocated`</td>
+    <td>Allocated amount of memory for this task</td>
+    <td>Bytes</td>
+    <td>Gauge</td>
+  </tr>
+  <tr>
+    <td>`nomad.client.allocs.<Job>.<TaskGroup>.<AllocID>.<Task>.disk.allocated`</td>
+    <td>Allocated amount of disk space for this task</td>
+    <td>Bytes</td>
+    <td>Gauge</td>
+  </tr>
+  <tr>
+    <td>`nomad.client.allocs.<Job>.<TaskGroup>.<AllocID>.<Task>.netwotk.<Device>.mbits.allocated`</td>
```

Comment:
@dadgar  you wrote that it's a lot of metrics and you are right, I'll try to move this to be emitted only when the allocation is placed, no need to report something that is not changed every 10 seconds